nextcloudappstore/scaffolding/app-templates/25/app/lib/AppInfo/Application.php

```diff
@@ -6,11 +6,30 @@
 namespace OCA\{{ app.namespace }}\AppInfo;
 
 use OCP\AppFramework\App;
+use OCP\AppFramework\Bootstrap\IBootContext;
+use OCP\AppFramework\Bootstrap\IBootstrap;
+use OCP\AppFramework\Bootstrap\IRegistrationContext;
 
-class Application extends App {
+class Application extends App implements IBootstrap {
 	public const APP_ID = '{{ app.id }}';
 
 	public function __construct() {
 		parent::__construct(self::APP_ID);
 	}
+
+    public function register(IRegistrationContext $context): void {
```

Comment:
yes, as Christophe mentioned, keeping indentation consistent will be helpful here.